helios.js

```diff
@@ -28652,6 +28709,98 @@ class InlineDatum extends Datum {
 	}
 }
 
+/**
+ * Encoding for Transcation Matadatum
+ */
+export class MetadataString extends CborData {
```

Comment:
I'm not sure that this is the best approach. How will we support things that are not strings? A special `from*` static method for each metadata type?

I was thinking to use type-inference of javascript types instead, so put statements like `if (typeof data == 'string') {...}` in `toCbor` instead (or even better: a `TxMetadata.encodeCbor` static method). Perhaps `Metadata` could be a jsdoc typedef?